mava/systems/ippo/networks.py

```diff
@@ -203,9 +216,16 @@ def make_discrete_networks(
         policy_layer_sizes: sizes of hidden layers for the policy network
         critic_layer_sizes: sizes of hidden layers for the critic network
         policy_recurrent_layer_sizes: Optionally add recurrent layers to the policy
-        recurrent_architecture_fn: Architecture to use for the recurrent units, e.g. LSTM or GRU.
-        policy_layers_after_recurrent: sizes of hidden layers for the policy network after the recurrent layers.
-        This is only used if an recurrent architecture is used.
+        recurrent_architecture_fn: Architecture to use for the recurrent units,
+        e.g. LSTM or GRU.
+        policy_layers_after_recurrent: sizes of hidden layers for the
+            policy network after the recurrent layers. This is only used if a
+            recurrent architecture is used.
+        orthogonal_initialisation: Whether network weights should be
```

Comment:
Would it perhaps be nicer to allow any initialiser to be passed in, or a dict of initialisers with keys such as "w_init", "b_init" etc? I'm not sure we need this flexibility, but just thought I would ask.

Reply:
I think it is something we could perhaps consider in the future? Since our current need is only to have orthogonal initialisation or not. Perhaps when we see a new for many different initialisers or want to experiment with it we can implement this. What are your thoughts?

Reply:
Yeah, makes sense, let's leave it for a future PR then, if we find we need the flexibility. 👍